### DISCUSSION TITLE
exploring the Integration of NextJS, WordPress Headless, and MDX

### DISCUSSION THREAD

**Discussion by wilonweb** [mdx-js/mdx#2431] [Q&A]
Hello everyone,

I'm currently diving into NextJS and learning how to use it with WordPress in headless mode. After getting more familiar with these, my next goal is to explore MDX.

I'm curious to know if anyone here has experience combining these technologies. Have you used MDX alongside a NextJS and headless WordPress setup? If so, I'd be really interested in hearing about your experiences, challenges, and any insights you might have on integrating these tools effectively.

**Comment by ChristianMurphy** [comment]
Welcome @wilonweb! 👋 

MDX is a language, it doesn't have opinions on what build tool or data source you use to feed it.
All of the pros and cons of this stack would be between NextJS and WordPress.
Also experiences, challenges, and insights would likely be based of _your use case_.
Which is unclear from what you mention in this ticket.

I'd encourage you @wilonweb to take some time to think about and articulate what you actually want these technologies to do. (see https://mdxjs.com/community/support/ for guidelines)
Also take a look through the existing Next and WordPress related discussions https://github.com/vercel/next.js/discussions?discussions_q=is%3Aopen+wordpress+headless
